pkg/systemd/terminal.html

```diff
@@ -23,7 +23,23 @@
     var term = null;
     var channel = null;
 
+    var current_user = cockpit.user;
+
+    var terminal_timer = null;
+    var shell_update_started = null;
     function start_terminal() {
+        if (shell_update_started === null)
```

Comment:
> Just if (!shell_update_started)

Again, I'd rather compare with null and avoid possible pitfalls with other values.